meta-resin-common/recipes-core/initrdscripts/files/machineid

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+machineid_enabled() {
+    mount -t ext4 /dev/disk/by-label/resin-state $ROOTFS_DIR/mnt/state
```

Comment:
While one would be foolish to use spaces in filenames, it is a legal character. Thus paths containing variables should be quoted.